### PR TITLE
Zanderz/core 2942

### PR DESCRIPTION
**Pull request by zanderz**
I took @chrisnojima 's hta (thanks for that!) and converted it to use plaintext arguments and registry results.
Some of the tests pass on Windows now, too, but not all.
I will check whether the other platforms still build before merging.
One idea I had was to pass the timeout to the .hta so it could close itself instead of being killed.
Anyway, once this is merged I can add it to the new installer. 
@gabriel

### CODE REVIEW

File: keybase/prompter/prompter.hta [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        window.resizeTo(screen.availWidth/2,screen.availHeight/2);
+        window.moveTo(screen.availWidth/4,screen.availHeight/4);
+     }
+       
+                  
+   </script>  
+<BODY>
+   <div class="relative">
+        <img class="centered_img" src="logo-with-text-dz2-sm.png" />
+
```
Comment (chrisnojima) [inline keybase/prompter/prompter.hta]: this isn't my hta???

Comment (zanderz) [inline keybase/prompter/prompter.hta]: Sorry about that, I checked in the wrong one!

**Comment by gabriel** [comment]
I'll take a look

File: keybase/platform_windows_test.go [hunk 2]
```diff
@@ -16,8 +16,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type testConfigWindows struct {
+	testConfigPlatform
+}
+
+func (c testConfigWindows) destinationPath() string {
+	return filepath.Join(os.Getenv("GOPATH"), "src/github.com/keybase/go-updater/windows/prompter/prompter.hta")
```
Comment (gabriel) [inline keybase/platform_windows_test.go]: Move this into test dir with other resources

File: keybase/platform_windows.go [hunk 3]
```diff
@@ -10,10 +10,22 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kardianos/osext"
 	"github.com/keybase/go-updater"
 	"github.com/keybase/go-updater/command"
 )
 
+// execPath returns the app bundle path where this executable is located
+func (c config) execPath() string {
```
Comment (gabriel) [inline keybase/platform_windows.go]: i think this is destination path? can move to that?

you could nuke execPath too unless we need it... if so it should be the path to the executable that started the app

File: keybase/platform_windows_test.go (hunk 2 above)
Comment (zanderz) [inline keybase/platform_windows_test.go]: This is the path to the production hta, which will be shipped - there isn't a test hta - where should it live?

File: keybase/platform_windows.go (hunk 3 above)
Comment (zanderz) [inline keybase/platform_windows.go]: The intent is for the updater to always find prompter.hta in a subdirectory named "prompter" under the current location from which upd.exe was run. Open to other suggestions of where it lives and how it gets found.

File: keybase/platform_windows_test.go (hunk 2 above)
Comment (gabriel) [inline keybase/platform_windows_test.go]: oh nm, this is real?

**Comment by gabriel** [comment]
We can nuke this in favor of other one?

**Pull request closed without merging**